tools/meson.build

```diff
@@ -50,16 +50,16 @@ tools = {
             'cndb_tests/cndb_txn_inject.c',
         ),
     },
-    'cndb_simpledrop_inject': {
-        'sources': files(
-            'cndb_tests/cndb_simpledrop_inject.c',
-        ),
-    },
-    'cndb_simpledrop_recovery_inject': {
-        'sources': files(
-            'cndb_tests/cndb_simpledrop_recovery_inject.c',
-        ),
-    },
+#    'cndb_simpledrop_inject': {
+#        'sources': files(
+#            'cndb_tests/cndb_simpledrop_inject.c',
+#        ),
+#    },
+#    'cndb_simpledrop_recovery_inject': {
+#        'sources': files(
+#            'cndb_tests/cndb_simpledrop_recovery_inject.c',
+#        ),
+#    },
```

Comment:
This is due to a case of checking in generated files and hence the resulting bitrot of the generators because they are not run as part of the build process.  I'll open a ticket...

Reply:
Which ticket is this?

Reply:
The one I've yet to open...

Reply:
Just ping me whenever you do.

Reply:
Will do, thanks!  I suspect you're they only one who can do it :-)

Reply:
https://jira.micron.com/jira/browse/NFSE-5152